src/main/resources/application.properties

```diff
@@ -1,10 +1,5 @@
 spring.application.name=BookStore
 
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimeZone=UTC
-spring.datasource.username=root
-spring.datasource.password=password
-
```

Comment:
better return variables for local running. you can use variables from env if you want to hide credentials